bin/before_install

```diff
@@ -6,6 +6,8 @@ APP_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/.." &>/dev/null && pwd)
 
 if [ -n "$CI" ]; then
   echo "== Installing system packages =="
+
+  sudo apt-get update
```

Comment:
Do we need a `-y`?
```suggestion
  sudo apt-get update -y
```

Reply:
No `apt-get update` doesn't prompt for anything